### ISSUE TITLE
Jobs not being en-queued on MSMQ, but in DEFAULT (Database table Job) instead.

### ISSUE THREAD

**Issue by iSatishYadav** [HangfireIO/Hangfire#1161]
**Jobs are not being en-queued on MSMQ, but in DEFAULT (Database table Job) instead.**
I can see the queue name on the list of queues, but Hangfire is not are not using MSMQ.

Do I need to specify the Queue name attribute on the Method Name?

**Package Details**
Hangfire.Core" version="1.6.17" targetFramework="net47"
Hangfire.SqlServer" version="1.6.17" targetFramework="net47"
Hangfire.SqlServer.MSMQ" version="1.6.17" targetFramework="net47"

**Configuration**
````
 GlobalConfiguration.Configuration
                .UseSqlServerStorage("MyConnectionString")
                .UseMsmqQueues(@"FormatName:Direct=OS:localhost\private$\background-{0}", "appname-default");

            app.UseHangfireDashboard();
            app.UseHangfireServer();
````

**Custom filters**
None.

**Application**
ASP.NET.

**Comment by pieceofsummer** [comment]
Yes, you need to specify Queue for the job method. Specifying queue names in MSMQ config only tells it which queues should be handled via MSMQ, not that all jobs should be enqueued to those queues.

You also need to specify your queues in server options when calling `app.UseHangfireServer()` for it to actually process them.
